src/app/passport/controllers/prove-another-way.js

```diff
@@ -0,0 +1,73 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const axios = require("axios");
+const logger = require("hmpo-logger").get();
+
+const {
+  API_BUILD_CLIENT_OAUTH_RESPONSE_PATH,
+  API_BASE_URL,
+} = require("../../../lib/config");
+
+class ProveAnotherWayController extends BaseController {
+  async saveValues(req, res, next) {
+    try {
+      logger.info("user submitting prove another way", { req, res });
+      req.sessionModel.set("redirect_url", undefined);
+
+      const action = req.form.values.proveAnotherWayRadio;
+
+      const headers = {
+        passport_session_id: req.session.passportSessionId,
+      };
+
+      switch (action) {
+        case "proveAnotherWay": {
```

Comment:
I might be understanding this wrong, but is the value for this not "continue"?

Reply:
I think this is because we have multiple possible steps in the form - for each section of the form we need to set up the requirements in `steps.js` so in this case it's
```
  "/prove-another-way": {
    prereqs: ["/passport/"],
    controller: proveAnotherWay, // this is the link to the case statement
    fields: ["proveAnotherWayRadio"],
    next: proveAnotherWay.prototype.next,
```
Lai’s solutions weren’t fully working until Robin explained about the `prereqs` attribute which allows this route of the form to be run if people have already gone to `/passport/`.